UM/View/SelectionPass.py

```diff
@@ -80,6 +98,24 @@ def render(self):
 
         self.release()
 
+    def _renderFacesMode(self):
+        batch = RenderBatch(self._face_shader)
+
+        selectable_objects = False
+        for node in Selection.getAllSelectedObjects():
+            if isinstance(node, ToolHandle):
```

Comment:
I think `node.isSelectable() and node.getMeshData()` this check should be enough. I don't know, but I don't think `ToolHandle` is selectable.